aria/cli/csar.py

```diff
@@ -22,7 +22,7 @@
 import requests
 from ruamel import yaml
 
-
+CSAR_FILE_EXTENSION = '.csar'
```

Comment:
do a quick search for `.csar`, there should be at least one more reference to it in modules you've touched anyway :)